cmd/vulcan-prowler/main.go

```diff
@@ -325,16 +363,31 @@ func fillCisLevelVuln(v *report.Vulnerability, r *prowlerReport, alias string, s
 			if err != nil {
 				return nil, err
 			}
+			cinfo, ok := controls[control]
+			if !ok {
+				return nil, fmt.Errorf("no information for control %s", control)
+			}
 			row := map[string]string{
-				"Control":     control,
-				"Description": description,
-				"Region":      e.Region,
-				"Message":     e.Message,
+				"Control":      control,
+				"Description":  description,
+				"CIS Severity": cinfo.SeverityLiteral,
+				"Region":       e.Region,
+				"Message":      e.Message,
+				"Remediation":  fmt.Sprintf("<a href=\"%s\">Reference</a>", cinfo.Remediation),
 			}
-			fcTable.Rows = append(fcTable.Rows, row)
+			c := controlRow{row, cinfo.Severity}
+			rows = append(rows, c)
+			fallthrough
+		default:
+			total++
 		}
 	}
-
+	sort.Slice(rows, func(i, j int) bool {
+		return rows[i].score > rows[j].score
+	})
+	for _, r := range rows {
+		fcTable.Rows = append(fcTable.Rows, r.row)
+	}
 	v.Resources = append(CISCompliance.Resources, fcTable)
```

Comment:
I know that this line is not part of this PR but I want to suggest a change:

Why not use the following form:
```
v.Resources = append(v.Resources, fcTable)
```

instead of:
```
v.Resources = append(CISCompliance.Resources, fcTable)
```
?

I am saying that because Golang docs for the `Append` function says that we should try to use the same variable as source and destination:

```
It is therefore necessary to store the result of append, often in the variable holding the slice itself
```
https://golang.org/pkg/builtin/#append

Again, this is just a suggestion.

Reply:
I completely agree thanks for the suggestion!!